src/DIRAC/RequestManagementSystem/Client/ReqClient.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, url=None, **kwargs):
         super(ReqClient, self).__init__(**kwargs)
         self.serverURL = "RequestManagement/ReqManager" if not url else url
 
-        self.log = gLogger.getSubLogger("RequestManagement/ReqClient/pid_%s" % (os.getpid()))
+        self.log = gLogger.getSubLogger(f"[pid_{os.getpid()}]{self.__class__.__name__}")
```

Comment:
out of curiosity, why now moving the class name after the pid ?

Reply:
I just followed the job id convention, but yeah it doesn't really make sense to apply it here. Do you want me to revert the change ?

Reply:
I think this specific one you can revert, and with this the PR would be ready for merge.